jsapp/js/account/plan.component.tsx

```diff
@@ -0,0 +1,442 @@
+          <label htmlFor='switch_right'> {t('Monthly')}</label>
+        </form>
+        <div
+          className={styles.currentPlan}
+          style={{
+            gridRow: 0,
+            gridColumn: 1 + filterPrices().findIndex(isSubscribedProduct),
+            display:
+              filterPrices().findIndex(isSubscribedProduct) >= 0 ? '' : 'none',
+          }}
+        >
+          {t('your plan')}
+        </div>
+        {filterPrices().map((price: Price, i: number) => (
+          <div className={styles.planContainer} key={i}>
+            <h1 className={styles.priceName}> {price.name} </h1>
+            <div className={styles.priceTitle}>
+              {typeof price.prices.human_readable_price === 'string' &&
+                (price.prices.human_readable_price.includes('$0.00')
+                  ? t('Free')
+                  : price.prices.human_readable_price)}
+            </div>
+
+            <ul>
+              {Object.keys(price.metadata).map(
+                (featureItem: string) =>
+                  featureItem.includes('feature_list_') && (
+                    <li key={featureItem}>
+                      <div className={styles.iconContainer}>
+                        <Icon
+                          name='check'
+                          size='m'
+                          classNames={
+                            price.name === 'Professional plan'
+                              ? [styles.tealCheck]
+                              : [styles.stormCheck]
+                          }
+                        />
+                      </div>
+                      {price.metadata[featureItem]}
+                    </li>
+                  )
+              )}
+            </ul>
+
+            {!isSubscribedProduct(price) && (
+              <button
+                className={[styles.resetButton, styles.upgradeBtn].join(' ')}
+                onClick={() => upgradePlan(price.prices.id)}
+                aria-label={`upgrade to ${price.name}`}
+                disabled={buttonsDisabled}
+                aria-disabled={buttonsDisabled}
+              >
+                {t('Upgrade')}
+              </button>
+            )}
+            {isSubscribedProduct(price) &&
+              state.organization?.uid &&
+              price.name !== 'Community plan' && (
+                <button
+                  className={[styles.resetButton, styles.manageBtn].join(' ')}
+                  onClick={managePlan}
+                  disabled={buttonsDisabled}
+                  aria-disabled={buttonsDisabled}
+                  aria-label={`manage your ${price.name} subscription`}
+                >
+                  {t('Manage')}
+                </button>
+              )}
+            {expandComparison && (
+              <div>
+                <div className={styles.line} />
+                {getListItem('support', price.name).length > 0 && (
+                  <ul>
+                    <li className={styles.listTitle}>
```

Comment:
It would be more semantic to not make the list title one of `<li>`s. See https://stackoverflow.com/questions/1141639/how-to-semantically-provide-a-caption-title-or-label-for-a-list-in-html